src/AcmProgressTracker/AcmProgressTracker.tsx

```diff
@@ -114,6 +125,19 @@ export function AcmProgressTracker(props: AcmProgressTrackerProps) {
                                     {step.statusSubtitle}
                                 </Text>
                             </TextContent>
+                            {step.link && (
+                                <AcmButton
+                                    className={classes.button}
+                                    variant="link"
+                                    icon={<ExternalLinkAltIcon size="sm" />}
+                                    iconPosition="right"
+                                    onClick={() => step.link?.linkTo()}
```

Comment:
@jamestalton  Any idea how to target this line in my testing? I am using `userEvent.click()` and it results in the callback being fired off, but jest is still finding that this line is not covered...